LayoutKit/Layouts/TextViewLayout.swift

```diff
@@ -0,0 +1,186 @@
+// Copyright 2016 LinkedIn Corp.
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+import UIKit
+
+/**
+ Layout for a UITextView.
+ */
+open class TextViewLayout<TextView: UITextView>: BaseLayout<TextView>, ConfigurableLayout {
+
+    // MARK: - instance variables
+
+    open let text: Text
+    open let font: UIFont?
+    open let textContainerInset: UIEdgeInsets
+    open let lineFragmentPadding: CGFloat
+
+    private lazy var isTextEmpty: Bool = {
```

Comment:
I'd make it a plain `let` property, since we probably always compute it once, don't need to be mutable, and would avoid any lazy (probably thread-safe) initialization code.